# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/apfs-fuse/hotfix-inttypes.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- apfs-fuse/ApfsLib/PList.h.vanilla	2025-11-18 17:42:00.720706618 +0200
+++ apfs-fuse/ApfsLib/PList.h	2025-11-18 17:42:10.858264130 +0200
@@ -3,6 +3,7 @@
 #include <vector>
 #include <map>
 #include <memory>
+#include <inttypes.h>
 
 enum PLType
 {
